### DISCUSSION TITLE
友人のGithubアカウントが見つからないです。

### DISCUSSION THREAD

**Discussion by ito-noizi-scratch** [misskey-dev/misskey#13628] [Q&A] [answered]
(私はGithubをはじめたばかりの中学生なので、文章の拙い部分もあると思います。指摘してくれると嬉しいです。)

友人をプライベートリポジトリに招待しようとしましたが、相手のアカウントが見つかりませんでした。
これは彼のプロフィールのリンクを共有してもらって飛んでも404エラーがでてしまったので、確かです。
また、彼が私をフォローしても表示されませんでした。

彼はプライベートプロフィールでもないようです。アカウントの設定を見てもらいました。
これが、そのスクリーンショットです。
![image](https://github.com/misskey-dev/misskey/assets/129924978/1edf9a4a-09b9-4ad7-bc77-c2268b9a6a75)

これの要因としては何が考えられるでしょうか？

P.S.
彼はアカウントを作ったのは1,2ヶ月まえですが、ほとんど動かしていませんでした。

**Comment by sonyakun** [comment, marked as answer]
友人のGitHubアカウントでログインした際などに「Your account has been flagged.
Because of that, your profile is hidden from the public. If you believe this is a mistake, contact support to have your account status reviewed.」と記載されたバナーが出ていますか？出ている場合はGitHubのサポートに英語などで問い合わせてください。原因を教えてくれると思います。

あとここはMisskeyのディスカッションなので、今度質問するときは[GitHub Community](https://github.com/orgs/community/discussions)で聞いてみたほうが良いかもしれません。
